src/message.rs

```diff
@@ -1355,6 +1355,14 @@ pub async fn get_msg_info(context: &Context, msg_id: MsgId) -> Result<String> {
             ret += &format!("\nLast seen as: {}/{}", server_folder, msg.server_uid);
         }
     }
+    let hop_info: Option<String> = context
+        .sql
+        .query_get_value("SELECT hop_info FROM msgs WHERE id=?;", paramsv![msg_id])
+        .await?;
+
+    if hop_info.is_some() {
+        ret.push_str(&hop_info.unwrap_or_else(|| "No Hop info".to_owned()));
```

Comment:
these lines add the hop-info to the massage-info

Reply:
That's down at the bottom though, but it's easy to move up to between "Sent:" and "Received:".

Reply:
ah, i missed that.  Are there any tests on this?  Sorry if my review skills lack a little lately but i can't find it in the PRs files.